app/views/documents/_attachment.html.erb

```diff
@@ -21,7 +21,7 @@
     <% end %>
   </div>
   <div class="attachment-details">
-    <h2 class="title"><%= link_to_unless previewable?(attachment), attachment.title, attachment.url(preview: preview), title_link_options %></h2>
+    <h3 class="title govuk-!-font-size-27 govuk-!-font-weight-regular"><%= link_to_unless previewable?(attachment), attachment.title, attachment.url(preview: preview), title_link_options %></h3>
```

Comment:
Taking at look a the temp [integration link](https://www.integration.publishing.service.gov.uk/government/publications/our-plan-to-rebuild-the-uk-governments-covid-19-recovery-strategy) this is as expected. ✅

---

🤔  Wondering if this diverges from the Design System a little and should just use default h3 styles. However out of scope

![image](https://user-images.githubusercontent.com/71266765/99528714-9bc2cc00-2996-11eb-96bb-d2baef3a5cef.png)

Reply:
That's a good point, imo it even looks nicer with the default styles but it's not within the scope of this piece of work to change it. 
We do have some exceptions from the default styling but I do wonder why this was one as I don't really see a benefit in it, in this particular case.